#!/usr/bin/env python
PACKAGE = "path_planner"

from dynamic_reconfigure.parameter_generator_catkin import *
#from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("leaf_size", double_t, 0, "Downsampling leaf size",    0.075, 0.0005,   0.2)
gen.add("density_radius_multiplier", double_t, 0, "denisty search radius",    5.0, 1.0,   20.0)

exit(gen.generate(PACKAGE, "path_planner", "TravAnalyzer"))

